logstash-core/lib/logstash/config/loader.rb

```diff
@@ -0,0 +1,99 @@
+require "logstash/config/defaults"
+
+module LogStash; module Config; class Loader
+  def initialize(logger, is_config_test)
+    @logger = logger
+    @is_config_test = is_config_test
```

Comment:
Hmm.. This flag (is_config_test) feels more like a flag controlling the debug level of the loader. If so, we can probably use the @logger.debug? method for that. Thoughts?